lib/spack/spack/detection/common.py

```diff
@@ -196,7 +196,10 @@ def find_win32_additional_install_paths():
     windows_search_ext.extend([os.environ[key] for key
                               in os.environ.keys() if
                               add_path(key)])
+    # Add search path for default Chocolately (https://github.com/chocolatey/choco)
+    # default install directory
```

Comment:
(minor) this says "default Chocolately default install directory"

* "default" appears twice
* I think it's called "Chocolatey"

As a side note, is this expected to be a good way to get CMake/Ninja? A short note in `getting_started` would be useful in that case.

Reply:
This is not the expected way to get Ninja or CMake (I am not aware of anyone who really uses chocolatey, just trying to cover our bases w.r.t. external find)

Reply:
Got it, in that case I think just updating the comment here is good.